site/src/docs/foundation/guidelines/server-side-rendering.mdx

```diff
@@ -0,0 +1,171 @@
+---
+slug: "/foundation/guidelines/server-side-rendering"
+title: "Server-side rendering"
+navTitle: 'Server-side rendering'
+---
+import { Accordion, Notification } from 'hds-react';
+import LeadParagraph from '../../../components/LeadParagraph';
+
+# Server-side rendering
+
+<LeadParagraph>
+  The server-side rendering support of the Helsinki Design System makes the user's landing experience to pages smooth without
+  flashes of unstyled content.
+</LeadParagraph>
+
+## What is server-side rendering?
+
+Server-side rendering (SSR) is the process where the rendering of HTML pages is done on the server-side. The fully rendered HTML
+document is then sent to the browser. The rendering of the HTML can happen at build time (Static site generation
+or pre-rendering), or during an HTTP request. The alternative to SSR is client-side rendering (CSR), where most of the HTML
+content is composed and rendered in the browser using JavaScript.
+
+## How does HDS support server-side rendering?
+
+For HDS components to work with server-side rendering, in server you need to inject the critical CSS styles
+of the HDS components that are being used into the initially rendered HTML. For hds-core, you must include the styles from
```

Comment:
"rendering, in **the** server" . Although a better structure might be to put the server mention in to the end (SPOTPA). Maybe something like this: "--rendering, you need to inject the critical CSS styles of the HDS components that are being used into the initially rendered HTML on the server's side."

Reply:
- [ ] rendering, in server you need to inject the critical CSS styles of the HDS components that are being used into the initially rendered HTML. -> rendering, you need to inject the critical CSS styles of the HDS components that are being used into the initially rendered HTML on the server's side.